utils/lxc/Makefile

```
@@ -57,7 +57,7 @@ endef
 define Package/lxc-auto
   $(call Package/lxc/Default)
   TITLE:= (initscript)
-  DEPENDS:=+lxc-start +lxc-stop
+  DEPENDS+=lxc-start +lxc-stop
```

```suggestion
  DEPENDS+=+lxc-start +lxc-stop
```
I had made this suggestion at the old commit, so it was not shown here in the PR.